packages/react-native-bridge/android/settings.gradle

```diff
@@ -22,11 +24,14 @@ pluginManagement {
 rootProject.name = '@wordpress_react-native-bridge'
 
 include ':react-native-bridge'
-include ':@wordpress_react-native-aztec'
-project(':@wordpress_react-native-aztec').projectDir = new File(rootProject.projectDir, '../../react-native-aztec/android')
-include ':react-native-aztec'
-project(':react-native-aztec').projectDir = new File(rootProject.projectDir, '../../../packages/react-native-aztec/android')
 
 if (hasProperty("willPublishReactNativeBridgeBinary")) {
     assert file("./react-native-bridge/build/assets/index.android.bundle").exists() : "index.android.bundle is necessary to publish a new version!"
+
+    include ':react-native-aztec'
+    project(':react-native-aztec').projectDir = new File(rootProject.projectDir, '../../../packages/react-native-aztec/android')
+}
+else {
+    include ':@wordpress_react-native-aztec'
+    project(':@wordpress_react-native-aztec').projectDir = new File(rootProject.projectDir, '../../react-native-aztec/android')
```

Comment:
Praise (❤️): Thanks for making this change to include the correct `react-native-aztec` project in bridge, this LGTM. PS: Maybe @oguzkocer can take a look and sign this off as well.
Minor (🔍): Consider having this `} else {` in a one-liner format.

Reply:
This suggestion was applied in https://github.com/WordPress/gutenberg/pull/52872/commits/bf340a441117768db07d5df889b0eae8ddb49c52.

Reply:
When we are publishing the `react-native-bridge` library, [we use the compiled artifact for `react-native-aztec`](https://github.com/WordPress/gutenberg/blob/rnmobile/0.69.4-upgrade-gradle-to-8.2.1-and-agp-to-8.0.2/packages/react-native-bridge/android/react-native-bridge/build.gradle#L122-L126), so we shouldn't need this conditional.

I think you if you remove the duplicate `react-native-aztec` project, it should work as expected. So, I suggest reverting the changes in this file and only remove the following:

```
include ':react-native-aztec'
project(':react-native-aztec').projectDir = new File(rootProject.projectDir, '../../../packages/react-native-aztec/android')
```

Reply:
> When we are publishing the `react-native-bridge` library, [we use the compiled artifact for `react-native-aztec`](https://github.com/WordPress/gutenberg/blob/rnmobile/0.69.4-upgrade-gradle-to-8.2.1-and-agp-to-8.0.2/packages/react-native-bridge/android/react-native-bridge/build.gradle#L122-L126), so we shouldn't need this conditional.

Good point. I'll remove including the Aztec project in the publishing case.

Reply:
This was fixed in https://github.com/WordPress/gutenberg/pull/52872/commits/66a02f94b91f429b77b2dc391bd41110dde00b38.